modal_proto/api.proto

```diff
@@ -1338,6 +1338,14 @@ message SandboxCreateResponse {
   string sandbox_id = 1;
 }
 
+message SandboxGetTaskIdRequest {
```

Comment:
I don't see the server PR for this up btw. We'll want to merge the proto definitions first, then deploy the server API change, and then the rest of this PR.